engine/worker/requirement.go

```diff
@@ -177,6 +158,11 @@ func checkServiceRequirement(w *currentWorker, r sdk.Requirement) (bool, error)
 }
 
 func checkMemoryRequirement(w *currentWorker, r sdk.Requirement) (bool, error) {
+	// If we are in a docker and in a provisioned worker don't run job in this worker
```

Comment:
Is it possible for the hatchery to add metadata or env variable with the memory ?  and so use provisioned worker?

Reply:
done